rules/windows/credential_access_posh_relay_tools.toml

```diff
@@ -0,0 +1,114 @@
+[metadata]
+creation_date = "2024/03/27"
+integration = ["windows"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2024/03/27"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects PowerShell scripts that can execute pass-the-hash (PtH) attacks, intercept and relay NTLM challenges, and carry
+out other man-in-the-middle (MitM) attacks.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.powershell*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Potential PowerShell Pass-the-Hash/Relay Script"
+references = [
+    "https://github.com/Kevin-Robertson/Invoke-TheHash/blob/master/Invoke-WMIExec.ps1",
+    "https://github.com/Kevin-Robertson/Invoke-TheHash/blob/master/Invoke-SMBExec.ps1",
+    "https://github.com/dafthack/Check-LocalAdminHash/blob/master/Check-LocalAdminHash.ps1",
+    "https://github.com/nettitude/PoshC2/blob/master/resources/modules/Invoke-Tater.ps1",
+    "https://github.com/Kevin-Robertson/Inveigh/blob/master/Inveigh.ps1"
+]
+risk_score = 47
+rule_id = "951779c2-82ad-4a6c-82b8-296c1f691449"
+setup = """## Setup
+
+The 'PowerShell Script Block Logging' logging policy must be enabled.
+Steps to implement the logging policy with Advanced Audit Configuration:
+
+```
+Computer Configuration >
+Administrative Templates >
+Windows PowerShell >
+Turn on PowerShell Script Block Logging (Enable)
+```
+
+Steps to implement the logging policy via registry:
+
+```
+reg add "hklm\\SOFTWARE\\Policies\\Microsoft\\Windows\\PowerShell\\ScriptBlockLogging" /v EnableScriptBlockLogging /t REG_DWORD /d 1
+```
+"""
+severity = "medium"
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Credential Access", "Resources: Investigation Guide", "Data Source: PowerShell Logs"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.category:process and host.os.type:windows and
```

Comment:
There is no `event.type == "start"` or something in these events?

Reply:
Hmmm there is `event.type == "info"`, maybe makes sense to push this in a separated PR, as none of the other PowerShell rules include it

Reply:
Think that would be good; especially giving the broad search of powershell text fields.

Reply:
Actually, I think we may not need to do that, as we specifically use the `logs-windows.powershell*` index